### ISSUE TITLE
Rule to enforce use of `$props` interface

### ISSUE THREAD

**Issue by AntonioVentilii** [sveltejs/eslint-plugin-svelte#1195]
### Description

Would it be possible to have a rule that enforces the explicit declaration of `Prop` interface?

```ts
// BAD
let {
	inputValue = $bindable(''),
	disabled = false,
	icon,
	overflowableContent
}: {
	inputValue: string;
	disabled?: boolean;
	icon: Snippet;
	overflowableContent?: Snippet;
} = $props()

// GOOD
interface Props {
	inputValue: string;
	disabled?: boolean;
	icon: Snippet;
	overflowableContent?: Snippet;
}

let {
	inputValue = $bindable(''),
	disabled = false,
	icon,
	overflowableContent
}: Props = $props()
```
